dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -140,30 +342,163 @@ export default defineComponent({
     };
   },
   name: "CreateEUTaxonomyForFinancials",
-  components: { DataPointFormElement, FailedUpload, FormKit, SuccessUpload, Card },
+  components: {
+    JumpLinksSection,
+    FailedUpload,
+    FormKit,
+    SuccessUpload,
+    UploadFormHeader,
+    Card,
+
+    UploadReports,
+    BasicInformationFields,
+
+    PrimeButton,
+    Calendar,
+    MultiSelect,
+    KPIfieldSet,
+  },
 
-  data: () => ({
-    innerClass: {
-      "formkit-inner": false,
-      "p-inputwrapper": true,
+  data() {
+    return {
+      formInputsModel: {} as CompanyAssociatedDataEuTaxonomyDataForFinancials,
+      filesToUpload: [] as ExtendedFile[],
+      uploadFiles: [] as ExtendedCompanyReport[],
+      fiscalYearEnd: "" as Date | "",
+      convertedFiscalYearEnd: "",
```

Comment:
I changed that now it can be null or Date and I change the names:
`      fiscalYearEndAsDate: null as Date | null,
      fiscalYearEnd: "",`